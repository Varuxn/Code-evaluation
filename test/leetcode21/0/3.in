10 10
aunxaofginlond
zuban
xbyrzxytnraoilwu
ltpbnezdfbsdvvluoxvlsqxxbifbljicxy
yhmjfeweeivqgcbkldjybcrpyqudu
jcravl
tuupxlxnoupcmchveuaqidzyzlbwdtweudxsneasawoydw
hjtekrwcz
lisifmnalqbwfuayoymsvfuefkvvuq
tvkp
yhmjfeweeivqgcbkldjybcrpyqudues sed oulzkmgoixkfcybaces xxhpfmjfwtking aunxaofginlonded tuupxlxnoupcmchveuaqidzyzlbwdtweudxsneasawoydwing hjtekrwczing kqpxjfgoqpnozerynbxsmldfvshogjiing wawhbisohjnyijidotyvwchring aunxaofginlondqwerqwe
