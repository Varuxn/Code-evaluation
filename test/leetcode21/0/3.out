yhmjfeweeivqgcbkldjybcrpyqudu sed oulzkmgoixkfcybaces xxhpfmjfwtking aunxaofginlond tuupxlxnoupcmchveuaqidzyzlbwdtweudxsneasawoydw hjtekrwcz kqpxjfgoqpnozerynbxsmldfvshogjiing wawhbisohjnyijidotyvwchring aunxaofginlond